website/docs/r/third_party_integration.markdown

```diff
@@ -12,8 +12,6 @@ description: |-
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
 
--> **NOTE:** Slack integrations now use the OAuth2 verification method and must be initially configured, or updated from a legacy integration, through the Atlas third-party service integrations page. Legacy tokens will soon no longer be supported.

```

Comment:
I'd actually leave this note, this explains how to now setup - otherwise people might think we left it off (we did for a reason). And still add the link 
[Atlas third-party service integrations page](https://docs.atlas.mongodb.com/tutorial/third-party-service-integrations/)

Reply:
Hi @themantissa I put the note back , and the link as well :)

Reply:
Thank you and extra bonus points for :) in the commit!